herd/variant.ml

```diff
@@ -55,6 +55,8 @@ type t =
   | ETS2 (*New feature introduced after deprecating ETS*)
 (* AArch64: Enhanced Exception Synchronization - FEAT_ExS *)
   | ExS | EIS | EOS
+  (* AArch64: Large Scale Extensions 2 - FEAT_LSE2 *)
```

Comment:
I think the name is Large System Extension ... (see [here](https://developer.arm.com/downloads/-/exploration-tools/feature-names-for-a-profile))
```suggestion
  (* AArch64: Large System Extensions version 2 - FEAT_LSE2 *)
```

Reply:
I've actually removed this. It doesn't make sense to have comments here when we have them in the mli file, does it?